src/Pos/Generator/Block/Mode.hs

```diff
@@ -123,7 +124,7 @@ data BlockGenContext = BlockGenContext
     , bgcSystemStart       :: !Timestamp
     , bgcParams            :: !BlockGenParams
     , bgcDelegation        :: !DelegationVar
-    , bgcGenesisUtxo       :: !GenesisUtxo
+    , bgcGenesisTxpContext :: !GenesisTxpContext
```

Comment:
It's better to remove. If later someone thinks they need it, they will have to think. And maybe it will turn out that it's not needed.